### PR TITLE
Feat: Create service with Dockerfile for meta extraction

### PR DESCRIPTION
**Pull request by carlosmondra**
This PR creates a new directory called services. The only service right now is meta-extraction which uses the meta extraction package. GCP will be configured for continuous deployment. This means that merging to master will deploy the latest Dockerfile.
In the future we can move the latex-validator to this repo too.

### CODE REVIEW

File: services/meta-extraction/src/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// Define routes
+app.get('/metadata', async (req, res) => {
+  if (!req.query.url) {
+    return res.status(400).json({
+      name: 'ValidationError',
+      message: 'URL is required',
+      status: 400
+    });
+  }
+  const { abstract, pdf, error } = await extractAbstract(req.query.url, !!req.query.skipTidy);
```
Comment (xkopenreview) [inline services/meta-extraction/src/index.js:20]: i think skipTidy won't be passed as query param
it should always be true

**Review by melisabok — APPROVED** [review]

File: services/meta-extraction/src/index.js (hunk 1 above)
Comment (xkopenreview) [reply services/meta-extraction/src/index.js:20]: sorry i mean it should always be false

Comment (carlosmondra) [reply services/meta-extraction/src/index.js:20]: Oh OK! I can make the change

Comment (carlosmondra) [reply services/meta-extraction/src/index.js]: DONE!

**Review by xkopenreview — APPROVED** [review]

**Pull request merged**